packages/venia-concept/src/components/MiniCart/product.css

```diff
@@ -13,9 +13,10 @@
 .image {
     border-radius: 2px;
     box-shadow: inset 0 0 0 1px rgb(var(--venia-border));
+    background-color: rgb(var(--venia-grey));
     grid-area: image;
-    height: 100px;
-    width: 80px;
+    background-position: 50% 50%;
+    background-size: cover;
```

Comment:
I guess I can try to find a plugin that will alphabetize CSS declarations. Is is not clear that they're all alphabetized?

Reply:
It was not, because I don't read letters, I read words. (Insert link to some kind of neuroscience article that absolves me here)